src/CalculatorUITests/StandardModeSmokeTests.cs

```diff
@@ -0,0 +1,92 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System;
+using CalculatorUITestFramework;
+
+namespace CalculatorUITests
+{
+    [TestClass]
+    public class StandardModeSmokeTests
```

Comment:
I don't think that we need to separate smoke tests from more thorough tests in the code. We currently have enough capacity to run all the tests all the time, and if we ever need to get more selective I would prefer that we do so by annotating individual tests by priority rather than having separate suites.

Reply:
We can leverage testcategory attributes to mark specific test cases if that's the direction we want to take it in.  

---
In reply to: [283480690](https://github.com/microsoft/calculator/pull/501#discussion_r283480690) [](ancestors = 283480690)